### DISCUSSION TITLE
Can not load large combo

### DISCUSSION THREAD

**Discussion by NoneAsName** [DrPython3/MailRipV2#2] [Q&A]
![image](https://user-images.githubusercontent.com/17923943/103639870-57dc5400-4f4f-11eb-98b3-fd3307e003bf.png)

Always fail with large combo,  I have checked combo and everything is fine with combo line.  My observation is that, sometimes it will stop on random line .

**Comment by DrPython3** [comment]
Hey there! Please check file encoding and set it to utf8 if any other one is set. I have tested combofiles with 500k, 1,500k and 2,000k of lines before first release without problems.

**Comment by DrPython3** [comment]
On the past weekend I began to add minor improvements to the code and could not find any line which may cause your problem except the file encoding. I have imported a public combolist with 1.7 million lines as a test. Though it took a long time, no error occurred. Another idea I had it could be depending on the OS being used. ATM I am only testing new releases on Linux (KALI) and macOS. Do you use Windows?

Best regards!

**Reply by jinxwoo13** [reply]
i'm using windows and i tried 90k combos it does failed the file is already encoded to utf8 idk why it keeps failing when i load huge combolists.

![Screenshot_2](https://github.com/DrPython3/MailRipV2/assets/93089612/dee8a6b6-6a81-4ba6-8a0a-50fa42b40611)
![Screenshot_1](https://github.com/DrPython3/MailRipV2/assets/93089612/0d73fa06-176c-46a4-aa2e-2eb84ba1b995)
